ports/planetd4rk/port.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": 3,
+    "name": "planetd4rk.zip",
+    "items": [
+        "Planet D4RK.sh",
+        "planetd4rk"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "Planet D4RK",
+        "porter": [
+            "tabreturn"
+        ],
+        "desc": "This game takes heavy inspiration from games like Celeste and Katana Zero. The game creator's speedrun PB is 5:17.399, so see if you can beat that time!",
+        "desc_md": null,
+        "inst": "Ready to run. Thanks to Abhi Sundu (https://abhimonk.itch.io/) for creating this great game and kindly PortMaster granting permission to distribute the game files.",
+        "inst_md": "Ready to run. Thanks to [Abhi Sundu](https://abhimonk.itch.io/) for creating this great game and kindly PortMaster granting permission to distribute the game files.",
+        "genres": [
+            "platformer"
+        ],
+        "image": {},
+        "rtr": true,
+        "exp": false,
+        "runtime": null,
+        "reqs": [],
+        "arch": [
+            "aarch64"
```

Comment:
This should be blank since it uses the included love 11.5 runtime which runs on all architectures. ❤️ 
```suggestion
```